### ISSUE TITLE
Code that intercepts XAML rule properties is conditioned on the .NET project capability.  Can we remove this dependency?

### ISSUE THREAD

**Issue by edwardskrod** [dotnet/project-system#7113]
**Visual Studio Version**:  16.10 Preview 2

**Summary**:
The Xaml Modern Apps Tools Team has added the Dependencies Node to the Windows Application packaging project by adding the DependenciesTree capability to the project file.  

We needed to add the .NET capability as well or we would get the following error:

```
LimitedFunctionality
System.AggregateException: Project system data flow 'ProjectEvaluationSubscriptionService' closed because of an exception.
There is no project properties provider for "Persistence = ProjectFileWithInterceptionViaSnapshot". ---> System.InvalidOperationException: There is no project properties provider for "Persistence = ProjectFileWithInterceptionViaSnapshot".
   at Microsoft.Verify.FailOperation(String message, Object[] args)
   at Microsoft.VisualStudio.ProjectSystem.PropertyPages.ProjectInstancePageRule.GetProjectPropertiesProvider(DataSource dataSource)
   at Microsoft.VisualStudio.ProjectSystem.PropertyPages.ProjectInstancePageRule.GetDataSourceConnector(DataSource dataSource, Boolean& pretranslatedPersistedName, String& defaultProjectPath)
   at Microsoft.VisualStudio.ProjectSystem.PropertyPages.PageRuleBase.GetPropertyValueAsync(String propertyName)
   at Microsoft.VisualStudio.ProjectSystem.Designers.ProjectEvaluationSubscriptionService.BuildPropertiesSnapshot(IRule rule, ImmutableDictionary`2 previousVersion, IImmutableSet`1& changedProperties)
```

This error popped up because we removed the .NET capability.   

According to @drewnoakes 

> Currently code that intercepts XAML rule properties is conditioned on the .NET project capability. This includes classes 
> ProjectFileWithInterceptionViaSnapshot, UserFileWithInterception and ProjectFileWithInterception.
> 
> Options are:
> 
> 1.	Reinstate the project capability (not ideal, as it lights up many other MEF components)
> 2.	Remove any properties that were added which use any of those three interception types
> 3.	Remove “.NET” from the AppliesTo expression on these three interception types 

@tmeschter  Do you have a view on this?

**Comment by drewnoakes** [comment]
FYI Tom is out for the rest of the week.

Did you investigate whether you can remove your intercepted properties? I suspect you may be able to, and that doing so would be the path of least resistance.

**Comment by drewnoakes** [comment]
@edwardskrod do you have an update on this? Were you able to remove your intercepted properties?

**Comment by edwardskrod** [comment]
I can look into this next sprint beginning Monday.  I have a few other high priority tasks, but I should be able to make time for this as well.

**Comment by edwardskrod** [comment]
@drewnoakes   This looks pretty straightforward.   Right off the bat, I can see that there are two offending properties that I added to ConfigurationGeneral - TargetFrameworkMonikers and TargetFrameworkMoniker.  Neither of these properties are necessary for our Dependency Tree to work properly.  I deleted them and will continue to work through removing the capability to see if there are any other intercepted properties that need removal.
